project1.py

```diff
@@ -0,0 +1,21 @@
+from bs4 import BeautifulSoup
+import requests
+#from urllib.request import urlopen
+import webbrowser
+import sys
+from fake_useragent import UserAgent
```

Comment:
https://travis-ci.com/TheAlgorithms/Python/builds/132468677#L334

Reply:
Please explain I don't understand what to do and how to do.

Reply:
I did't get error

Reply:
Comment out the line that it is complaining about and then see if the script still runs.  flake8 is saying that the import is not needed.

Reply:
> Comment out the line that it is complaining about and then see if the script still runs. flake8 is saying that the import is not needed.

commented